ci-operator/config/kubevirt/must-gather/kubevirt-must-gather-main.yaml

```diff
@@ -56,7 +56,8 @@ tests:
       cli: latest
       commands: |
         oc adm must-gather --image="${KMG_IMAGE}" --dest-dir=${ARTIFACT_DIR}/must-gather-output
-        (cd ${ARTIFACT_DIR}; KUBECONFIG=${KUBECONFIG} ACK_GINKGO_DEPRECATIONS=1.16.4 /go/bin/must-gather.test --ginkgo.focus=level:product --junit-output=${ARTIFACT_DIR}/junit/report.xml --ginkgo.v)
+        mkdir -p ${ARTIFACT_DIR}/junit
+        (cd ${ARTIFACT_DIR}; KUBECONFIG=${KUBECONFIG} /go/bin/must-gather.test --ginkgo.label-filter=level:product --ginkgo.junit-report=${ARTIFACT_DIR}/junit/report.xml --ginkgo.v)
```

Comment:
since you are changing directory to `${ARTIFACT_DIR}`, does it make sense for `ginkgo.junit-report` to be set to `${ARTIFACT_DIR}/junit/report.xml` and not only `junit/report.xml`